composer.json

```diff
@@ -6,6 +6,11 @@
     "require-dev": {
         "friendsofphp/php-cs-fixer": "^3.14"
     },
+    "config": {
```

Comment:
I added because if I try to update the dependencies using my php 8.0, It won't be compatible with php7 anymore. Adding this config help `composer update` only target php >=7.4.